ldclient/impl/datasourcev2/__init__.py

```diff
@@ -1,6 +1,79 @@
 """
 This module houses FDv2 types and implementations of synchronizers and
 initializers for the datasystem.
+
+All types and implementations in this module are considered internal
+and are not part of the public API of the LaunchDarkly Python SDK.
+They are subject to change without notice and should not be used directly
+by users of the SDK.
+
+You have been warned.
 """
 
+from abc import abstractmethod
```

Comment:
Moving the shared types to this top level. In my next bit of work, I'm going to be re-arranging the polling stuff, so just focus on streaming for now.